src/code/models/codap-connect.coffee

```diff
@@ -77,10 +79,18 @@ module.exports = class CodapConnect
       # ret==null is indication of timeout, not an indication that the data set
       # doesn't exist.
       if !ret or ret.success
+        @_initialSyncAttributeProperties ret.values?.collections?[1].attrs
```

Comment:
Are you missing an existential operator after the [1] like this:

`@_initialSyncAttributeProperties ret.values?.collections?[1]?.attrs`